rewards/df-guides.md

```diff
@@ -0,0 +1,39 @@
+---
+description: Show you can complete all User-Guides as the last steps to master Data Farming!
+---
+
+# Complete User Guides
+
+<figure><img src="../.gitbook/assets/gif/neo-kinda-martial-arts.gif" alt=""><figcaption><p>Like Neo, you have great potential.</p></figcaption></figure>
+
+### Get Started
+
+Our [User Guides](../user-guides/data-farming/README.md) get you started on Data Farming quickly so you can complete all operations. Follow these guides to get a complete handle on all Data Farming activities.
+
+{% content-ref url="../user-guides/data-farming/how-to-veocean.md" %}
+[data-farming/how-to-veocean.md](../user-guides/data-farming/how-to-veocean.md)
```

Comment:
All the links linking to user guides are broken :(

Reply:
Please see comments below & use the 2nd gitbook link.

I have reviewed all links, gifs, and others in this section, and they all seem to be working. Resolving this convo.